### ISSUE TITLE
[Higher Level Review]: Focus doesn't go to the h3, but progress bar

### ISSUE THREAD

**Issue by davidakennedy** [department-of-veterans-affairs/va.gov-team#83632]
## What happened?

**Describe what went wrong**:

When focus gets shifted from the 

**Add the steps to reproduce the bug**:

1. Log in with a test user account. For example: `vetsgov+test236@id.me`.
2. Get to the "Confirm the personal information we have on file for you." screen. Fill out a choice, and navigate to the next screen. Notice how focus gets moved to the progress bar, but not the h3 of the next question.

This is inconsistent with how the rest of the form works. Ideally, focus should shift to the h3 "Are you experiencing homelessness?"

**Add a screen recording or photo**:

<img width="1361" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK0g4QkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--72df8bae973e28f569d7097d08913e43dd99f266/Screenshot%202024-05-21%20at%201.47.29%E2%80%AFPM.png" alt="The Are you experiencing homelessness? screen on the HLR with a red arrow pointing to the console showing focus going to the progress bar." />

## How severe is the problem?

- [ ] **Critical**: User is block or significantly hindered from submitting the form, data is lost, or claim and evidence test coverage is missing. Fix ASAP.
- [ ] **Moderate**: User is not blocked, but the happy path is interrupted. Fix after critical bugs.
- [x] **Minor**: Cosmetic bugs and minor language inconsistencies. Defer to a later sprint.

## What is the desired behavior?

1. _When a user does X, the **visual** result is y._
2. _When a user does X, the **backend** result is y._
3. _When a user does X, the **event logging** result is y._

## Definition of Done

- [ ] Acceptance criteria reviewed and approved by product or design
- [ ] Unit test (or other test coverage) added if possible
- [ ] Logging and monitoring in place

**Comment by Mottie** [comment]
@davidakennedy This work is now available in staging and ready for validation. There is one PR associated with this ticket still pending, but it's for the [`/form-saved` page focus on alert H2](https://github.com/department-of-veterans-affairs/vets-website/pull/30066).
